code/modules/integrated_electronics/subtypes/reagents.dm

```diff
@@ -16,59 +16,6 @@
 	set_pin_data(IC_OUTPUT, 1, reagents.total_volume)
 	push_data()
 
-/obj/item/integrated_circuit/reagent/smoke
```

Comment:
why did you delete this?

Reply:
The circuit nerf from a few weeks ago removed smoke circuits entirely, while this PR still had the code of the smoke circuit in it, since this was written even before the nerf. As such, this would have overwritten the nerf and reintroduced smoke circuits, hence the merge conflict.